<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>321</width>
     <height>279</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="ipaddress_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="toolTip">
           <string>IP Address</string>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="port_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Window Title</string>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="gspro_window_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>GSPro Window Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="gspro_api_window_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>GSPro API Window Title</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPlainTextEdit" name="gspro_path_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Application Path(Opyional)</string>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::WidgetWidth</enum>
            </property>
            <property name="placeholderText">
             <string>GSPro Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="file_browse_button">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Default Device</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="default_device_combo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>New Item</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Launch Monitor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="launch_monitor_combo">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>New Item</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="toolTip">
         <string>New Device</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_button">
        <property name="toolTip">
         <string>New Device</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
